src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/servicebus/ServiceBusHelper.java

```diff
@@ -69,7 +75,17 @@ Message mapToBusMessage(Msg msg) {
         busMessage.setMessageId(msg.getMsgId());
         busMessage.setMessageBody(getMsgBodyInBytes(msg));
         busMessage.setLabel(msg.getLabel());
-
+        busMessage.setProperties(new HashMap<String,String>());
+        RequestTelemetryContext context = ThreadContext.getRequestTelemetryContext();
+        String parentId;
+        if (context != null
+            && context.getHttpRequestTelemetry() != null
+            && context.getHttpRequestTelemetry().getId() != null) {
+            parentId = context.getHttpRequestTelemetry().getId();
+        } else {
+            parentId = LocalStringsUtils.generateRandomIntegerId();
```

Comment:
do we have to create `HttpRequestTelemetry` if `context.getHttpRequestTelemetry()` is null ?

Reply:
it is in other pr. when tasks run we will create telemetry request by AOP.
https://github.com/hmcts/bulk-scan-processor/pull/1336